Camera2Extensions/app/src/main/java/com/example/android/camera2/extensions/fragments/SelectorFragment.kt

```diff
@@ -99,9 +102,18 @@ class SelectorFragment : Fragment() {
                                           .REQUEST_AVAILABLE_CAPABILITIES_BACKWARD_COMPATIBLE) &&
                 extensionCharacteristics.supportedExtensions.isNotEmpty()) {
                 availableCameras.add(CameraInfo("$orientation ($id)", id))
+                val currentExtension = extensionCharacteristics.supportedExtensions[0]
```

Comment:
We might need to scan across all supported extensions to see which one supports jpeg r

Reply:
Done - initially I thought the sample app hardcoded setting the extension for the first available hence this, but I see that it switches through them in the CameraFragment so I updated to iterate through all JPEG_R supporting extensions there as well